### PR TITLE
Fix type errors in order app v2

### PR DESCRIPTION
**Pull request by ds300**
✔️

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -57,15 +57,15 @@
...
+    "@babel/core": "^7.7.2",
     "@babel/plugin-proposal-class-properties": "7.3.4",
     "@babel/plugin-proposal-decorators": "7.3.0",
     "@babel/plugin-proposal-json-strings": "7.2.0",
     "@babel/plugin-syntax-dynamic-import": "7.2.0",
     "@babel/plugin-syntax-import-meta": "7.0.0",
     "@babel/preset-env": "7.3.4",
     "@babel/preset-react": "7.0.0",
-    "@babel/preset-typescript": "7.3.3",
+    "@babel/preset-typescript": "^7.7.2",
```
Comment (alloy) [inline package.json]: Can you remind me why this was needed?

Comment (alloy) [reply package.json]: I see, so it’s because of syntax needed in stories.

Comment (ds300) [reply package.json]: `as const`

**Comment by artsyit** [comment]
:rocket: PR was released in v22.0.0 :rocket:

**Pull request merged**